express/src/views/layouts/base.njk

```diff
@@ -13,6 +13,8 @@
 
 {% set govUkGridClass = gridClass or "govuk-grid-column-two-thirds" %}
 
+{% set govUkGridClass = gridClass or "govuk-grid-column-two-thirds" %}
```

Comment:
This line is not needed as it duplicates line 14

Reply:
I think this was a mistake on merging: I'll remove it.